lib/delayed/performable_method.rb

```diff
@@ -23,7 +24,13 @@ def display_name
     end
 
     def perform
-      object.send(method_name, *args) if object
+      return unless object
+
+      if kwargs.nil? || (RUBY_VERSION < '2.7' && kwargs.empty?)
```

Comment:
You can check if the callee uses kwargs. First get the method object with `object.method(method_name)` and check its parameter list for `:key`, `:keyreq` or `:keyrest`. (From experience, Rails Mailer magical methods end up with a single `:rest` argument that you'd need to account for as well.)

Reply:
In this case, I don't think we want it to actually care about the arity of the method it's about to call -- only the arity that was provided to it by the enqueuing call. (For example, if I do `obj.delay.something(test: 123)` on a method that _does not accept kwargs_, I want to ensure that the job raises an error and tells me that my arguments are wrong.)

So the fix here is really just to prevent `**{}` from breaking on Ruby 2.6 (because trying to pass that as a kwarg results in it passing `{}` as a positional arg). This quirk was fixed in 2.7.